docs/how-to/use-layers.md

```diff
@@ -0,0 +1,147 @@
+# How to use layers
+
+Managing multiple services across different environments becomes complex as systems scale. Pebble simplifies this with layered configurations, improving readability and maintainability.
+
+A base layer defines common settings (such as logging), while additional layers handle specific services or environment-specific overrides. This declarative approach, along with delegated layer management, allows for better cross-team collaboration and provides a clear view of each environment's configuration. For example, an operations team could manage base layers for logging, and service teams could manage layers for their services.
+
+## Pebble layers
+
+A layer is a configuration file that defines the desired state of services running on a system. 
+
+Layers are organized within a `layers/` subdirectory in the `$PEBBLE` directory. Their filenames are similar to `001-base-layer.yaml`, where the numerically prefixed filenames ensure a specific order of the layers, and the labels after the prefix uniquely identifies the layer. For example, `001-base-layer.yaml`, `002-override-layer.yaml`.
+
+A layer specifies service properties including the command to execute, startup behavior, dependencies on other services, and environment variables. For example:
+
+```yaml
+summary: Simple layer
+
+description: |
+    A better description for a simple layer.
+
+services:
+    srv1:
+        override: replace
+        summary: Service summary
+        command: cmd arg1 "arg2a arg2b"
+        startup: enabled
+        after:
+            - srv2
+        before:
+            - srv3
+        requires:
+            - srv2
+            - srv3
+        environment:
+            VAR1: val1
+            VAR2: val2
+            VAR3: val3
+
+    srv2:
+        override: replace
+        startup: enabled
+        command: cmd
+        before:
+            - srv3
+
+    srv3:
+        override: replace
+        command: cmd
+```
+
+For full details of all fields, see [layer specification](../reference/layer-specification).
+
+## Layer override
+
+Each layer can define new services or modify existing ones defined in preceding layers. The mandatory `override` field in each service definition determines how the layer's configuration interacts with the previously defined service of the same name (if any):
+
+- `override: replace` completely replaces the previous definition of a service.
+- `override: merge` combines the current layer's settings with the existing ones, allowing for incremental modifications.
+
+Any of the fields can be replaced individually in a merged service configuration. See the [full layer specification](../reference/layer-specification) for details.
```

Comment:
I think it'd be good to show a very simple override layer here, that maybe adds one environment variable to `server`, and then show the merged result. Something like:

```
services:
  server:
    override: replace
    environment:
        PORT: 8081
```

The final environment from the combined (merged) layers will be:

```
DATABASE=dbserver.example.com
PORT=8081
```

Or similar.